acceptance_testing.go

```diff
@@ -1124,5 +1123,60 @@ func (a acceptanceTest) context(t *testing.T) context.Context {
 	return ctx
 }
 
+func (a acceptanceTest) configureSource(ctx context.Context, t *testing.T, src Source) error {
+	return configureSource(
```

Comment:
Same as with `configureDestination`. I wonder if we could leave this either as a method or as a function.